.github/actions/get-tools-versions/action.yml

```diff
@@ -46,6 +46,10 @@ runs:
       run: |
         # Default R version
         echo "r=latest" >> $GITHUB_OUTPUT
```

Comment:
(not related to this PR): I would move the lines 47 and 48 as _ELSE-part_ to 
```
          if [[ "$val1" == "R" ]]; then
            echo "r=$val2" >> $GITHUB_OUTPUT
          fi
```

Reply:
Not sure exactly how bash `while` loops work, would an if-else statement create issues if not on the last line of the csv ?